homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -31,6 +32,12 @@ class PlugwiseBinarySensorEntityDescription(BinarySensorEntityDescription):
 
 
 BINARY_SENSORS: tuple[PlugwiseBinarySensorEntityDescription, ...] = (
+    PlugwiseBinarySensorEntityDescription(
+        key="low_battery",
+        translation_key="low_battery",
+        device_class=BinarySensorDeviceClass.BATTERY,
```

Comment:
Should we just fallback to the device class provided translations?

Reply:
That would mean the name would become `battery`.
We already have a sensor called `battery` showing the state of charge in %.
I found that too confusing, that's why I named this type of binary_sensor as `battery_state`.

Reply:
But that would break stuff 😮‍💨

Reply:
In that case I would like to keep the name as is :)